src/styles.css

```diff
@@ -1,6 +1,18 @@
 * {
   font-family: 'Roboto', sans-serif;
-  /* border: solid 1px green; */
+}
+
+button:focus, .search-field:focus, .food-category:focus, 
+.selectMenu:focus, .quantityInput:focus{
+  outline-width: 4px;
+  outline-style: solid;
+  outline-color: #3c40c6;
+}
+
+figure:focus{
```

Comment:
I didn't how to do this! Looks great Kiko! Thank you so much!

Reply:
Also didn't know how to do this. You're an accessibility wizard